pkgs/development/libraries/gmock/default.nix

```diff
@@ -1,15 +1,13 @@
-{ stdenv, fetchurl, unzip, cmake}:
-
+{ stdenv, cmake, callPackage }:
+let
+  source = callPackage ./source.nix { };
```

Comment:
@vcunat Done (using fetchzip).

Didn't know about fetchzip. Now I do :-). I didn't implement the fetch, I just split out the maintainer's fetching logic into a separate output so I didn't investigate too deeply.